Mathlib/Analysis/SpecialFunctions/Exp.lean

```diff
@@ -117,6 +117,37 @@ theorem ContinuousOn.cexp (h : ContinuousOn f s) : ContinuousOn (fun y => exp (f
 theorem Continuous.cexp (h : Continuous f) : Continuous fun y => exp (f y) :=
   continuous_iff_continuousAt.2 fun _ => h.continuousAt.cexp
 
+/--The complex exponential function is uniformly continuous on left half planes. -/
+lemma UniformlyContinuosOn_cexp (a : ℝ) : UniformContinuousOn exp {x : ℂ | x.re ≤ a} := by
+  have : Continuous (cexp - 1) := Continuous.sub (Continuous.cexp continuous_id') continuous_one
+  rw [Metric.uniformContinuousOn_iff, Metric.continuous_iff'] at *
+  intro ε hε
+  simp only [gt_iff_lt, Pi.sub_apply, Pi.one_apply, dist_sub_eq_dist_add_right,
+    sub_add_cancel] at this
+  have ha : 0 < ε / (2 * Real.exp a) := by positivity
+  have H := this 0 (ε / (2 * Real.exp a)) ha
+  rw [Metric.eventually_nhds_iff] at H
+  obtain ⟨δ, hδ⟩ := H
+  refine ⟨δ, hδ.1, ?_⟩
+  intros x _ y hy hxy
+  have h3 := hδ.2 (y := x - y) (by simpa only [dist_zero_right, norm_eq_abs] using hxy)
+  rw [dist_eq_norm, exp_zero] at *
+  have : cexp x - cexp y = cexp y * (cexp (x - y) - 1) := by
+      rw [@mul_sub_one, ← exp_add]
```

Comment:
Is the @ needed?

Reply:
no that was just copilot! I didnt notice it :)